atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/transaction/impl/SnapshotTransaction.java

```diff
@@ -1349,9 +1348,7 @@ private <T> ListenableFuture<Collection<Map.Entry<Cell, T>>> getWithPostFilterin
 
         getCounter(AtlasDbMetricNames.SNAPSHOT_TRANSACTION_CELLS_READ, tableRef).inc(rawResults.size());
 
-        // LinkedList is chosen for fast append operation since we just add to this collection.
-        @SuppressWarnings("JdkObsolete")
-        Collection<Map.Entry<Cell, T>> resultsAccumulator = new LinkedList<>();
```

Comment:
I'm guessing we haven't enabled JdkObsolete error prone check due to the heavy `Sorted*` interface usages, but that would have flagged this https://errorprone.info/bugpattern/JdkObsolete

Reply:
I think we did do this in https://github.com/palantir/atlasdb/pull/5682 (otherwise there probably wouldn't be the suppression here).

The `JdkObsolete` check has lots of exceptions for various things. I assume the remaining uses just fall under those exceptions.